benchmarks/smallmatvec/manual_versions/openmp/matrix_vector_kernel_mod.F90

```diff
@@ -368,7 +382,9 @@ subroutine matrix_vector_code_nlayersf_split(cell,        &
 end subroutine matrix_vector_code_nlayersf_split
 
 
-
+! Like the original but manually blocking the outer loop with blocks of size
+! 8 (vlen variable) to match the vector length. This this blocks are iterated
```

Comment:
s/This this blocks are iterated/The members of each block are then iterated over in an inner loop/

Reply:
Done